client/src/components/contact-map/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {useEffect} from "react";
+import { ContactMapInfo, ContactMapSection } from "./contact-map";
+import { YMaps, Placemark } from "react-yandex-maps";
+import { Container } from "../../box/absolute-box";
+import AOS from "aos";
+import "aos/dist/aos.css";
+
+
+const ContactMap = () => {
+  useEffect(() => {
+    AOS.init({ duration: 2000 });
+  }, []);
+
+  const coordinates = [35.1815784, -120.7474753];
```

Comment:
move this constant above component. we don't need initializate this on every render.

Reply:
thank you i corrected